consent-ws/pom.xml

```diff
@@ -215,6 +215,27 @@
 
     <dependencies>
 
+
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-api</artifactId>
+            <version>0.10.1</version>
```

Comment:
Latest version here is `0.10.5` ... there have been a bunch of releases in August.